_posts/0000-01-13-martinquenneville.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "This is Martin Quenneville's slide deck!"
 ---
 
 ![chill-out-lemur](https://cloud.githubusercontent.com/assets/16547949/25400761/9c64ee82-29c1-11e7-8ad3-6886f97634b9.jpg)
 {: .center}
 
-Caption here
+Don't worry, I've got this!
```

Comment:
@martinquenneville Are you sure people will understand the 2nd degree of the joke ?